test/unit/Init.t.sol

```diff
@@ -60,9 +64,10 @@ contract InitTest is SetUp {
         withPY(address(this), 1000000 ether)
     {
         InitParams memory initParams = getDefaultParams();
+        PYIndex index = IPYieldToken(PT.YT()).newIndex();
```

Comment:
FYI I've added a `newIndex()` function to the SetUp so we don't need to import all the PY related things anymore.